WebRoot/js/zimbraMail/mail/controller/ZmMailListController.js

```diff
@@ -387,6 +387,7 @@ function(actionCode, ev) {
 					} else {
 						frag = item.fragment ? item.fragment : ZmMsg.fragmentIsEmpty;
 						if (frag != "") { lv.setToolTipContent(AjxStringUtil.htmlEncode(frag), true); }
```

Comment:
This looks better, Thanks.
Updated PR: https://github.com/Zimbra/zm-web-client/pull/59/commits/1fe5ff5d46b408ee52263b547986021f984dc6ef